crates/client/rpc/src/versions/v0_7_1/methods/read/get_block_with_receipts.rs

```diff
@@ -11,7 +11,7 @@ pub fn get_block_with_receipts(
     starknet: &Starknet,
     block_id: BlockId,
 ) -> StarknetRpcResult<MaybePendingBlockWithReceipts> {
-    log::debug!("block_id {block_id:?}");
+    tracing::debug!("block_id {block_id:?}");
```

Comment:
let's update the comment here

Reply:
Updated to 
```tracing::debug!("get_block_with_receipts called with {:?}", block_id);```
[here](https://github.com/madara-alliance/madara/pull/348/commits/7da86a8fc2fc83f7da37ba8b78344a66f7863657).

Reply:
Updated [here](https://github.com/madara-alliance/madara/pull/348/commits/7da86a8fc2fc83f7da37ba8b78344a66f7863657).